.github/workflows/bootstrap-plan-apply-component.yaml

```
@@ -152,14 +67,15 @@ jobs:
       run: terraform fmt -check -recursive
 
     - name: Check Terragrunt Formatting
+      working-directory: ${{ steps.workdir.outputs.directory }}
```

actionlint (1.7.8)

property "workdir" is not defined in object type {}

(expression)